myia/compile/backends/__init__.py

```diff
@@ -20,46 +22,87 @@ class LoadingError(Exception):
     """
 
 
-def backend_loader(pkg, name):
-    """Helper function for simple backends.
+class BackendLoader:
+    """Utility class to load a backend."""
 
-    This will return a callable that will load a module, retrieve a
-    object from its namespace and return that.
+    __slots__ = ("load_options", "load_backend")
```

Comment:
Why does this have a `__slots__` declaration?  It doesn't harm anything but I'm curious.

Reply:
Sorry, an habit ! `__slots__` helps to lock list of allowed attributes and help python to reduce memory footprint (by denying `__dict__` and allocate only fields declared in slots).

But it is useless here, I will remove it.

Reply:
I knew what it does, but I was just curious why it was here.

Reply:
Ok ! I removed it anyways !